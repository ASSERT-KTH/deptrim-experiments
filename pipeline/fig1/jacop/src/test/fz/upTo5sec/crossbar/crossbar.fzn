array [1..2] of int: X_INTRODUCED_1 = [1,-1];
var {2,4,6,7,8,9,10,11,16,18,20}: V1:: output_var;
var {2,3,4,8,10,12,17,19,20}: V2:: output_var;
var {2,3,4,6,8,9,11,17,18}: V3:: output_var;
var {1,3,4,5,6,7,9,10,11,13,18}: V4:: output_var;
var {1,5,6,10,12,13,14,17,18,19,20}: V5:: output_var;
var {1,3,10,12,15,16,19,20}: V6:: output_var;
var {5,8,9,10,17}: V7:: output_var;
var {1,2,5,6,7,12,14,15,16,17}: V8:: output_var;
var {1,2,3,4,5,7,11,12,13,14,16,17,20}: V9:: output_var;
var {4,5,8,9,10,11,13,17,18,19,20}: V10:: output_var;
var {2,4,6,7,8,10,12,14,17,18,20}: V11:: output_var;
var {3,7,8,9,10,13,14,15,18,20}: V12:: output_var;
var {2,3,6,7,8,9,11,13,16,20}: V13:: output_var;
var {2,3,5,6,8,9,12,13,15,16,17,18}: V14:: output_var;
var {2,7,8,10,12,13,14,15,16,17,18,20}: V15:: output_var;
var {1,2,6,11,13,16,17,19,20}: V16:: output_var;
var {1,3,6,9,13,19}: V17:: output_var;
var {1,3,6,7,8,10,13,14,19}: V18:: output_var;
var {1,2,3,4,5,6,7,9,11,12,14,16,17,19,20}: V19:: output_var;
var {3,5,6,7,8,9,11,12,13,14,16,18,20}: V20:: output_var;
array [1..20] of var int: List = [V1,V2,V3,V4,V5,V6,V7,V8,V9,V10,V11,V12,V13,V14,V15,V16,V17,V18,V19,V20];
constraint int_lin_ne(X_INTRODUCED_1,[V1,V2],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V3],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V4],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V5],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V6],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V7],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V8],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V9],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V10],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V1,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V3],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V4],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V5],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V6],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V7],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V8],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V9],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V10],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V2,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V4],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V5],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V6],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V7],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V8],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V9],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V10],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V3,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V5],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V6],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V7],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V8],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V9],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V10],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V4,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V6],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V7],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V8],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V9],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V10],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V5,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V7],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V8],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V9],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V10],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V6,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V8],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V9],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V10],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V7,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V9],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V10],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V8,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V10],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V9,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V11],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V10,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V11,V12],0);
constraint int_lin_ne(X_INTRODUCED_1,[V11,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V11,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V11,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V11,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V11,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V11,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V11,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V11,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V12,V13],0);
constraint int_lin_ne(X_INTRODUCED_1,[V12,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V12,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V12,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V12,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V12,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V12,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V12,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V13,V14],0);
constraint int_lin_ne(X_INTRODUCED_1,[V13,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V13,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V13,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V13,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V13,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V13,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V14,V15],0);
constraint int_lin_ne(X_INTRODUCED_1,[V14,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V14,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V14,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V14,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V14,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V15,V16],0);
constraint int_lin_ne(X_INTRODUCED_1,[V15,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V15,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V15,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V15,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V16,V17],0);
constraint int_lin_ne(X_INTRODUCED_1,[V16,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V16,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V16,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V17,V18],0);
constraint int_lin_ne(X_INTRODUCED_1,[V17,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V17,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V18,V19],0);
constraint int_lin_ne(X_INTRODUCED_1,[V18,V20],0);
constraint int_lin_ne(X_INTRODUCED_1,[V19,V20],0);
solve  satisfy;
