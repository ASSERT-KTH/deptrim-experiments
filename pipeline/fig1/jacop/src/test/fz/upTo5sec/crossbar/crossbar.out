V1 = 10;
V2 = 2;
V3 = 4;
V4 = 11;
V5 = 13;
V6 = 3;
V7 = 5;
V8 = 12;
V9 = 20;
V10 = 17;
V11 = 14;
V12 = 8;
V13 = 9;
V14 = 15;
V15 = 16;
V16 = 6;
V17 = 1;
V18 = 7;
V19 = 19;
V20 = 18;
----------